packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -147,8 +147,15 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
 
   @override
   Future<LostData> retrieveLostData() async {
-    final Map<String, dynamic>? result =
-        await _channel.invokeMapMethod<String, dynamic>('retrieve');
+    Map<String, dynamic>? result;
+    List<PickedFile>? pickedFileList = [];
+    bool isRetrieve = false;
+    try {
+      result = await _channel.invokeMapMethod<String, dynamic>('multiRetrieve');
+    } on UnimplementedError {
```

Comment:
It is actually a MissingPluginException
```suggestion
    } on MissingPluginException {
```